packages/discord.js/src/structures/PermissionOverwrites.js

```diff
@@ -181,7 +181,13 @@ class PermissionOverwrites extends Base {
     }
 
     const userOrRole = guild.roles.resolve(overwrite.id) ?? guild.client.users.resolve(overwrite.id);
-    if (!userOrRole) throw new DiscordjsTypeError(ErrorCodes.InvalidType, 'parameter', 'User nor a Role');
+    if (!userOrRole) {
+      throw new DiscordjsTypeError(
+        ErrorCodes.InvalidType,
+        'parameter',
+        'could not be resolved to a cached User or Role',
+      );
```

Comment:
Just for the record, this will generate this message:

> Supplied parameter is not a could not be resolved to a cached User or Role.

https://github.com/discordjs/discord.js/blob/c484e3de250e2ed52f9651b1fc7f2b27990ef632/packages/discord.js/src/errors/Messages.js#L75